src/components/sbb-form-field/sbb-form-field.stories.js

```diff
@@ -0,0 +1,428 @@
+    </select>
+  </sbb-form-field>
+);
+
+const TemplateSelectWithError = (args) => (
+  <sbb-form-field {...args}>
+    <select slot="input" class="input select sbb-invalid" placeholder="Name">
+      <option>Value 1</option>
+      <option>Value 2</option>
+      <option>Value 3</option>
+    </select>
+    <sbb-form-error error-space={args['error-space']} id="error" slot="error">
+      This is a required field.
+    </sbb-form-error>
+  </sbb-form-field>
+);
+
+const TemplateSelectWithErrorSpace = (args) => {
+  const sbbFormError = (
+    <sbb-form-error
+      error-space={args['error-space']}
+      id="error-1"
+      className="input-label-error"
+      slot="error"
+    >
+      This is a required field.
+    </sbb-form-error>
+  );
+
+  return (
+    <form>
+      <sbb-form-field id="sbb-form-field" {...args}>
+        <select
+          id="sbb-form-field-input"
+          onFocus={(event) => {
+            if (event.currentTarget.value === '') {
+              document.getElementById('sbb-form-field').append(sbbFormError);
+              document
+                .getElementById('sbb-form-field-input')
+                .setAttribute('class', 'input sbb-invalid form-field--size-l');
+            }
+          }}
+          onBlur={(event) => {
+            if (event.currentTarget.value !== '') {
+              sbbFormError.remove();
+              document
+                .getElementById('sbb-form-field-input')
+                .setAttribute('class', 'input form-field--size-l');
+            }
+          }}
+          slot="input"
+          class="input sbb-invalid"
+          placeholder="Name"
+        >
+          <option>{''}</option>
+          <option>Value 1</option>
+          <option>Value 2</option>
+          <option>Value 3</option>
+        </select>
+        {sbbFormError}
+      </sbb-form-field>
+      <br />
+      <sbb-form-field {...args}>
+        <input slot="input" class="input" placeholder="Name" />
+      </sbb-form-field>
+    </form>
+  );
+};
+
+const TemplateSelectWithIcons = (args) => (
+  <sbb-form-field {...args}>
+    <span slot="prefix" class="input__icon-prefix">
+      <sbb-icon name="pie-small" />
+    </span>
+    <select slot="input" class="input select" placeholder="Name">
+      <option>Value 1</option>
+      <option>Value 2</option>
+      <option>Value 3</option>
+    </select>
+    <span slot="suffix" class="input__icon-suffix">
+      <sbb-icon name="circle-information-small" />
+    </span>
+  </sbb-form-field>
+);
+
+const disabledArg = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const errorSpaceArg = {
+  control: {
+    type: 'select',
+  },
+  options: ['default', 'reserve'],
+};
+
+const labelArg = {
+  control: {
+    type: 'text',
+  },
+};
+
+const optionalArg = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const readonlyArg = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const sizeArg = {
+  control: {
+    type: 'inline-radio',
+  },
+  options: ['l', 'm'],
```

Comment:
The default/first otion should be `m`. `l` is only intended for special use cases.